docs/understanding-tracking-design/managing-event-specifications/ui/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Managing Event Specifications in the Console"
+sidebar_label: "Using the UI"
+sidebar_position: 1
+sidebar_custom_props:
+  offerings:
+    - enterprise
+    - cloud
+---
+
+## Crafting a New Event Specification through the Console
```

Comment:
Change to 
'Creating a new event specification through Console'

Reply:
done